src-docs/src/views/popover/popover_example.js

```diff
@@ -349,5 +361,30 @@ export const PopoverExample = {
       snippet: popoverFixedSnippet,
       demo: <PopoverFixed />,
     },
+    {
+      title: 'Popover as display block',
```

Comment:
Reads a little more englishy™

```suggestion
      title: 'Popover with block level display',
```